src/components/summary.json

```diff
@@ -24,6 +24,25 @@
       ]
     },
     "graphConfig": {
+      "useDemo": true,
```

Comment:
I'm assuming I'll deploy this branch to only to cosri-demo, but would it make sense to make `useDemo` conditional on an environment variable (if from `env.json`) or a specific test patient name?

Reply:
@ivan-c I fixed the code so it will look for environment variable, `USE_DEMO`, to check if demo data should be used for the graph.  I hope that works?